package/MDAnalysis/core/groups.py

```diff
@@ -2203,55 +2203,19 @@ class AtomGroup(GroupBase):
     :class:`AtomGroup` instances are always bound to a
     :class:`MDAnalysis.core.universe.Universe`. They cannot exist in isolation.
 
-    .. rubric:: Deprecated functionality
-
-    *Instant selectors* will be removed in the 1.0 release.  See issue `#1377
-    <https://github.com/MDAnalysis/mdanalysis/issues/1377>`_ for more details.
-
-    :class:`Atoms<Atom>` can also be accessed in a Pythonic fashion by using the
-    :class:`Atom` name as an attribute. For instance, ::
-
-        ag.CA
-
-    will provide an :class:`AtomGroup` of all CA :class:`Atoms<Atom>` in the
-    group. These *instant selector* attributes are auto-generated for
-    each atom name encountered in the group.
-
-    Notes
-    -----
-    The name-attribute instant selector access to :class:`Atoms<Atom>` is mainly
-    meant for quick interactive work. Thus it either returns a single
-    :class:`Atom` if there is only one matching :class:`Atom`, *or* a
-    new :class:`AtomGroup` for multiple matches.  This makes it difficult to use
-    the feature consistently in scripts.
-
 
     See Also
     --------
     :class:`MDAnalysis.core.universe.Universe`
 
-
     .. deprecated:: 0.16.2
        *Instant selectors* of :class:`AtomGroup` will be removed in the 1.0
-       release. See :ref:`Instant selectors <instance-selectors>` for details
-       and alternatives.
+       release.
+    .. versionchanged:: 1.0
```

Comment:
A bit of a pedantic comment, but should `versionchanged` also include the patch version?

Reply:
Generally yes

Reply:
yes